### DISCUSSION TITLE
v3.0.5

### DISCUSSION THREAD

**Discussion by carhartl** [js-cookie/js-cookie#820] [General]
* Remove npm version restriction in package.json - #818

<hr /><em>This discussion was created from the release <a href='https://github.com/js-cookie/js-cookie/releases/tag/v3.0.5'>v3.0.5</a>.</em>

**Comment by igitur** [comment]
Hi @carhartl . Would you mind adding the build output like with the previous releases?
![image](https://user-images.githubusercontent.com/145854/235130850-20da7b98-2d9c-444d-ab3c-b8bb0660e13a.png)

thanks.

**Comment by carhartl** [comment]
I stopped this, as this counteracts package provenance I started releasing with. You can use `npm pack` to download the package contents.

**Comment by igitur** [comment]
Ok. I maintain the [NuGet package](https://nuget.org/packages/js-cookie), although not very diligently. This built output was just convenient for me update the package quickly. I think I'll to v3.0.4 for one last time. Then people should move to npm for this.
